spec/features/conference_registration_spec.rb

```diff
@@ -49,5 +65,16 @@
         expect(conference.user_registered?(participant)).to be(true)
       end
     end
+
+    context 'registration is closed' do
+      let(:conference_with_closed_registration) { create(:conference, registration_period: create(:registration_period)) }
+
+      scenario 'registers for a conference', feature: true do
+        participant.is_admin = false
```

Comment:
So it is either this or, I make a user to throw away like other specs. I think we shouldn't have to do either.. hence the issue.